webapi/src/main/scala/org/knora/webapi/slice/admin/api/StoreEndpoints.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2021 - 2024 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.slice.admin.api
+
+import sttp.tapir.*
+import sttp.tapir.generic.auto.*
+import sttp.tapir.generic.auto.schemaForCaseClass
+import sttp.tapir.json.zio.jsonBody
+import sttp.tapir.query
+import zio.ZLayer
+import zio.json.DeriveJsonCodec
+import zio.json.JsonCodec
+
+import org.knora.webapi.messages.store.triplestoremessages.RdfDataObject
+import org.knora.webapi.messages.store.triplestoremessages.RdfDataObject.jsonCodec
+import org.knora.webapi.slice.common.api.BaseEndpoints
+
+final case class MessageResponse(message: String)
+object MessageResponse {
+  implicit val jsonCodec: JsonCodec[MessageResponse] = DeriveJsonCodec.gen[MessageResponse]
+}
+
+final case class StoreEndpoints(baseEndpoints: BaseEndpoints) {
```

Comment:
There is `StoresRestService` but `StoreEndpoints` and ` StoreEndpointsHandler`. I think the last two (including filenames) should be aligned with rest service, which follows the plural naming convention.